lona/client/dom-updater.js

```diff
@@ -467,7 +474,16 @@ Lona.LonaDomUpdater = function(lona_context, lona_window) {
 
             // REMOVE
             } else if(operation == protocol.OPERATION.REMOVE) {
-                node.removeAttribute(data[0]);
+                var name = data[0];
+
+                // properties
+                if(property_names.indexOf(name) > -1) {
+                    node[name] = false;
+
+                // attributes
+                } else {
+                    node.removeAttribute(name);
+                };
 
             // CLEAR
```

Comment:
Don't we need to do the same check on clear?

Reply:
Ack! I will fix that.